cg/meta/report/api.py

```diff
@@ -120,6 +184,12 @@ def _get_case_from_statusdb(self, case_id: str) -> models.Family:
         """Fetch a case object from the status database."""
         return self.store.family(case_id)
 
+    def _get_analysis_from_statusdb(
```

Comment:
Should this not be defined in store as a general sub to import and reuse whenever you need to fetch an analysis. Together with a nice unit test and not as a utility sub here?

Reply:
I agree, but this PR is to old to keep alive for that, lets merge it while it is still a bit warm